package.json

```diff
@@ -46,10 +47,8 @@
     "ts-node": "10.9.1",
     "tslint": "5.5.0",
     "typescript": "^4.7.4",
-    "vsce": "1.103.1",
-    "vscode-test": "1.6.1",
-    "webpack": "^5.74.0",
-    "webpack-cli": "^4.10.0"
+    "vsce": "2.11.0",
+    "vscode-test": "1.6.1"
```

Comment:
bump the vsce version. I'm not sure why we were pinned to the old version, but I verified the versions in the release notes and there are no breaking changes. The major version bump was for upgrading the versions of node that are supported.